cmd/notation/cert/add.go

```diff
@@ -28,6 +28,14 @@ func certAddCommand(opts *certAddOpts) *cobra.Command {
 			opts.path = args
 			return nil
 		},
+		Long: `Manage certificates in trust store
+
+Example - Add certificates with type "ca" to the trust store "acme-rockets":
+  notation cert add --type ca --store "acme-rockets" $XDG_CONFIG_HOME/notation/truststore/x509/ca/acme-rockets/
```

Comment:
`$XDG_CONFIG_HOME/notation/truststore/x509/ca/acme-rockets/` is the trust store. It should be changed to certificates file user wants to add, for example "./acme-rockets.crt

Reply:
Updated